src/view/com/util/List.web.tsx

```diff
@@ -303,14 +377,15 @@ let Visibility = ({
 
   React.useEffect(() => {
     const observer = new IntersectionObserver(handleIntersection, {
+      root,
       rootMargin: `${topMargin} 0px ${bottomMargin} 0px`,
     })
```

Comment:
The default for `IntersectionObserver` is for `root` to be the viewport. This works fine whenever we are not using container mode, but when we are we want our intersections to be determined based on the container's scroll area, not the full window's.

See: https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/root